samples/sample-app/server/server.js

```diff
@@ -21,25 +21,15 @@ const startServer = (port = 3000, PUBLIC_KEY, PRIVATE_KEY, URL_ENDPOINT) => {
                 urlEndpoint: URL_ENDPOINT
             });
 
-    
-            router.get("/auth", (req, res) => {
+            router.get("/", (req, res) => {
                 try {
+                    // Generating security parameters.
```

Comment:
Setting `token`, `expiration` and `signature`  on initial HTML response is not a good way. It would always fail on second upload request. Please call an API from the frontend using Javascript.